stdlib/public/Python/PythonLibrary.swift

```diff
@@ -115,26 +133,37 @@ private extension PythonLibrary {
     }
     
     func set(_ value: String) {
+      #if canImport(Darwin) || canImport(Glibc)
       setenv(key, value, 1)
+      #elseif os(Windows)
+      _putenv_s(key, value)
+      #endif
     }
   }
 }
 
 // Methods of `PythonLibrary` required to load the Python library.
 private extension PythonLibrary {
   static let supportedMajorVersions: [Int] = [3, 2]
-  static let supportedMinorVersions: [Int?] = [nil] + Array(0...30).reversed()
-
+  static let supportedMinorVersions: [Int] = Array(0...30).reversed()
+  
```

Comment:
Remove trailing spaces.

Reply:
You mean in line 149? All the file contains indent spaces in empty lines (this is the default behavior in Xcode).

Reply:
This file originally did not have trailing whitespaces, but this PR added some.  We usually turn on "Automatically trim trailing whitespace" - "Including whitespace-only lines" in Xcode.

Reply:
No, originally it had them. In any case, I have removed them to unify the style with the rest of the module.

Reply:
Sorry, I was think of Python.swift. Thanks for removing them.